packages/teams-js/test/private/conversations.spec.ts

```diff
@@ -32,6 +34,19 @@ describe('conversations', () => {
       );
     });
 
+    it('should throw error if not supported in runtime config', async () => {
```

Comment:
I think it would be good to include the details like "conversation capability is not supported", etc. for these.